provider/completeness.md

```diff
@@ -0,0 +1,1070 @@
+| Resource | Readme | Docs | Acceptance Test | Acceptance File | Resource File | Unit test |
+| --- | --- | --- | --- | --- | --- | --- |
+|      databricks_aws_assume_role_policy | ✅ | ❌ (0%) | ❌ (0%) | ❌ | ❌ | ❌ (0%) |
+|           databricks_aws_bucket_policy | ✅ | ❌ (0%) | ❌ (0%) | ❌ | ❌ | ❌ (0%) |
+|     databricks_aws_crossaccount_policy | ✅ | ❌ (0%) | ❌ (0%) | ❌ | ❌ | ❌ (0%) |
+|                databricks_aws_s3_mount | ❌ | ❌ (0%) | ❌ (0%) | ✅ | ❌ | ❌ (0%) |
+|       databricks_azure_adls_gen1_mount | ❌ | ❌ (0%) | ❌ (0%) | ❌ | ❌ | ❌ (0%) |
+|       databricks_azure_adls_gen2_mount | ❌ | ❌ (0%) | ❌ (0%) | ❌ | ❌ | ❌ (0%) |
+|            databricks_azure_blob_mount | ❌ | ❌ (0%) | ❌ (0%) | ✅ | ❌ | ❌ (0%) |
+|                     databricks_catalog | ✅ | ✅ (100%) | ✅ (100%) | ❌ | ✅ | ✅ (100%) |
+|                    databricks_catalogs | ✅ | ❌ (0%) | ❌ (0%) | ❌ | ❌ | ❌ (0%) |
+|                     databricks_cluster | ✅ | ✅ (98%) | ✅ (98%) | ✅ | ✅ | ✅ (98%) |
+|              databricks_cluster_policy | ✅ | ✅ (100%) | ✅ (100%) | ✅ | ✅ | ✅ (100%) |
+|                    databricks_clusters | ✅ | ❌ (0%) | ❌ (0%) | ❌ | ❌ | ❌ (0%) |
+|                databricks_current_user | ✅ | ❌ (0%) | ❌ (0%) | ❌ | ❌ | ❌ (0%) |
+|                   databricks_dbfs_file | ✅ | ✅ (83%) | ✅ (83%) | ✅ | ✅ | ✅ (83%) |
+|                   databricks_dbfs_file | ✅ | 👎 (75%) | ✅ (75%) | ✅ | ✅ | ✅ (75%) |
+|             databricks_dbfs_file_paths | ✅ | ❌ (0%) | ❌ (0%) | ❌ | ❌ | ❌ (0%) |
+|                   databricks_directory | ❌ | ✅ (100%) | ✅ (100%) | ❌ | ✅ | ✅ (100%) |
+|           databricks_external_location | ✅ | ✅ (85%) | ✅ (85%) | ❌ | ✅ | ✅ (85%) |
+|              databricks_git_credential | ❌ | ✅ (100%) | ✅ (100%) | ✅ | ✅ | ✅ (100%) |
+|          databricks_global_init_script | ✅ | ✅ (83%) | ✅ (83%) | ✅ | ✅ | ✅ (83%) |
+|                      databricks_grants | ✅ | ✅ (100%) | ✅ (100%) | ❌ | ✅ | ✅ (100%) |
+|                       databricks_group | ✅ | ✅ (87%) | ✅ (87%) | ✅ | ✅ | ✅ (87%) |
```

Comment:
It would be useful to add an icon or text to distinguish between resources and data sources, otherwise it's hard to understand what is it

Reply:
will do

Reply:
i'm thinking how even generate this file